### PR TITLE
Fixed linting errors

### PR DESCRIPTION
**Pull request by rob2244**
Fixed all of the linting errors, fixed most of the warnings, but some of them are not worth fixing

### CODE REVIEW

File: src/DBMetadata/metadataRetriever.ts [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
 
-interface IMetadataRetriever {
+interface MetadataRetriever {
     getSchemaInfo(connString: string): Promise<DBMetadata[]>;
     buildConnectionString(info: ConnData): string;
 }
 
-export default IMetadataRetriever
\ No newline at end of file
+export default MetadataRetriever
```
Comment (laueian) [inline src/DBMetadata/metadataRetriever.ts]: new line

Comment (rob2244) [reply src/DBMetadata/metadataRetriever.ts]: ?

File: src/Generators/provider/dbProvider.ts [hunk 2]
```diff
@@ -9,4 +9,4 @@ interface IDBProvider {
     configureExport(): void;
 }
 
-export default IDBProvider
\ No newline at end of file
+export default DBProvider
```
Comment (laueian) [inline src/Generators/provider/dbProvider.ts]: new line

File: src/DBMetadata/metadataRetriever.ts (hunk 1 above)
Comment (laueian) [reply src/DBMetadata/metadataRetriever.ts]: There should always be a new line at the end of the a file

**Comment by laueian** [comment]
Looks good, can you add the new lines then squash your commits please

File: src/DBMetadata/metadataRetriever.ts (hunk 1 above)
Comment (rob2244) [reply src/DBMetadata/metadataRetriever.ts]: Why?

Comment (laueian) [reply src/DBMetadata/metadataRetriever.ts]: https://evanhahn.com/newline-necessary-at-the-end-of-javascript-files/

Comment (laueian) [reply src/DBMetadata/metadataRetriever.ts]: I'll add it to the formatter so we won't have to worry about it ourselves, so you don' t need to fix it if you don't want to

**Pull request merged**